cmd/zetae2e/local/local.go

```diff
@@ -318,6 +314,10 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 			e2etests.TestBitcoinWithdrawP2SHName,
 			e2etests.TestBitcoinWithdrawP2WSHName,
 			e2etests.TestBitcoinWithdrawRestrictedName,
+			e2etests.TestBitcoinStdMemoDepositName,
```

Comment:
consider creating an issue for moving them back after the upgrade

Reply:
As mentioned on Slack, no need if we should push this for v21.
The test should pass once we set v21 as the reference for the upgrade tests